Doxyfile

```diff
@@ -559,7 +553,7 @@ INTERNAL_DOCS          = NO
 # names in lower-case letters. If set to YES, upper-case letters are also
 # allowed. This is useful if you have classes or files whose names only differ
 # in case and if your file system supports case sensitive file names. Windows
-# (including Cygwin) and Mac users are advised to set this option to NO.
+# (including Cygwin) ands Mac users are advised to set this option to NO.
```

Comment:
typo?
```suggestion
# (including Cygwin) and Mac users are advised to set this option to NO.
```

Reply:
This is how Doxygen generated the Doxyfile. I left it as is.... Looks like this string is still in their codebase. https://github.com/doxygen/doxygen/blob/master/src/config.xml#L953